docs/source/jointhecommunity.rst

```diff
@@ -6,3 +6,11 @@ SQLFluff Slack
 We have a fast-growing community on Slack, come and join us!
 
 https://join.slack.com/t/sqlfluff/shared_invite/zt-o1f4x0e8-pZzarAIlQmKj_6ZwD16w0g
```

Comment:
```suggestion
We have a fast-growing community [on Slack](https://join.slack.com/t/sqlfluff/shared_invite/zt-o1f4x0e8-pZzarAIlQmKj_6ZwD16w0g), come and join us!
```

Reply:
Yeah could inline this link - consistent with the README. However it's an .rst file and not markdown to syntax is different. Will add separately.